azext_edge/edge/vendor/clients/deviceregistryrefreshmgmt/_client.py

```diff
@@ -0,0 +1,180 @@
+    :ivar schema_registries: SchemaRegistriesOperations operations
+    :vartype schema_registries: deviceregistry.mgmt.operations.SchemaRegistriesOperations
+    :ivar namespace_assets: NamespaceAssetsOperations operations
+    :vartype namespace_assets: deviceregistry.mgmt.operations.NamespaceAssetsOperations
+    :ivar namespace_devices: NamespaceDevicesOperations operations
+    :vartype namespace_devices: deviceregistry.mgmt.operations.NamespaceDevicesOperations
+    :ivar namespace_discovered_assets: NamespaceDiscoveredAssetsOperations operations
+    :vartype namespace_discovered_assets:
+     deviceregistry.mgmt.operations.NamespaceDiscoveredAssetsOperations
+    :ivar namespace_discovered_devices: NamespaceDiscoveredDevicesOperations operations
+    :vartype namespace_discovered_devices:
+     deviceregistry.mgmt.operations.NamespaceDiscoveredDevicesOperations
+    :ivar schemas: SchemasOperations operations
+    :vartype schemas: deviceregistry.mgmt.operations.SchemasOperations
+    :ivar schema_versions: SchemaVersionsOperations operations
+    :vartype schema_versions: deviceregistry.mgmt.operations.SchemaVersionsOperations
+    :param credential: Credential needed for the client to connect to Azure. Required.
+    :type credential: ~azure.core.credentials.TokenCredential
+    :param subscription_id: The ID of the target subscription. The value must be an UUID. Required.
+    :type subscription_id: str
+    :param endpoint: Service URL. Default value is None.
+    :type endpoint: str
+    :keyword api_version: Api Version. Default value is "2025-07-01-preview". Note that overriding
+     this default value may result in unsupported behavior.
+    :paramtype api_version: str
+    :keyword int polling_interval: Default waiting time between two polls for LRO operations if no
+     Retry-After header is present.
+    """
+
+    def __init__(
+        self, credential: "TokenCredential", subscription_id: str, endpoint: Optional[str] = None, **kwargs: Any
+    ) -> None:
+        _cloud = kwargs.pop("cloud_setting", None) or settings.current.azure_cloud  # type: ignore
+        _endpoints = get_arm_endpoints(_cloud)
+        if not endpoint:
+            endpoint = _endpoints["resource_manager"]
+        credential_scopes = kwargs.pop("credential_scopes", _endpoints["credential_scopes"])
+        self._config = MicrosoftDeviceRegistryManagementServiceConfiguration(
+            credential=credential, subscription_id=subscription_id, credential_scopes=credential_scopes, **kwargs
+        )
+
+        _policies = kwargs.pop("policies", None)
+        if _policies is None:
+            _policies = [
+                policies.RequestIdPolicy(**kwargs),
+                self._config.headers_policy,
+                self._config.user_agent_policy,
+                self._config.proxy_policy,
+                policies.ContentDecodePolicy(**kwargs),
+                ARMAutoResourceProviderRegistrationPolicy(),
```

Comment:
This may be a newer version of autorest components than other generated clients.

Reply:
yea - I figured it wasnt worth spending 50 years trying to debug why autorest hates everyone like so:
![image](https://github.com/user-attachments/assets/8859e037-8392-4f90-b4d9-bcf03a773328)

also this will be regenerated once you get the approved arm swagger